spec/systems/categories_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :system do
+  user = User.new(id: 1, name: 'Saeqa', email: 'saeqa@gmail.com', password: '123456', confirmed_at: Time.now)
+
+  group = Group.create!(user_id: user.id, name: 'test', icon: 'burger.png')
```

Comment:
- [ ] [REQUIRED] In the cause of trying to fix the error, I think you have broken some lines of code @SaeqaSultani . 
- We now have an error that states User does not exist.

<img width="594" alt="Screenshot 2023-04-29 at 23 52 10" src="https://user-images.githubusercontent.com/58692590/235327384-386e8cb0-1c3c-42e7-b3a6-fc6a8884af33.png">

Recommendation:
I think we need to remove the `confirmed_at` attribute from the User instantiation to avoid the UnknownAttributeError. Take a look at what your code should look like;
``` ruby
user = User.new(id: 1, name: 'Saeqa', email: 'saeqa@gmail.com', password: '123456')
```
But Incase the error persists, Ensure a valid User object exists: Before creating the Group object, verify that a valid User object is available. You can create a User object explicitly or use an existing User object if available.
``` ruby
user = User.create!(name: 'Saeqa', email: 'saeqa@gmail.com', password: '123456')
group = Group.create!(user_id: user.id, name: 'test', icon: 'burger.png')
```